components/expenses/ExpenseForm.js

```diff
@@ -67,11 +69,16 @@ const msg = defineMessages({
     id: 'ExpenseForm.StepExpenseFundingRequest',
     defaultMessage: 'Set grant details',
   },
-
   stepPayee: {
     id: 'ExpenseForm.StepPayeeInvoice',
     defaultMessage: 'Payee information',
   },
+  resetExpense: {
+    defaultMessage: 'Reset Form',
+  },
+  confirmResetExpense: {
+    defaultMessage: 'Do you really want to reset the expense form?',
```

Comment:
A small suggestion; not a blocker. I would rather use `Are you sure...` for consistency. That's how most of our confirmations are worded in the front-end. Also `Do you really....` gives the impression that its a negative action, something not desirable. 😺 

```suggestion
    defaultMessage: 'Are you sure you want to reset the expense form?',
```

Reply:
Makes sense, updated 👍🏻